app/Actions/Photo/Strategies/AddStandaloneStrategy.php

```diff
@@ -0,0 +1,164 @@
+<?php
+
+namespace App\Actions\Photo\Strategies;
+
+use App\Contracts\SizeVariantFactory;
+use App\Contracts\SizeVariantNamingStrategy;
+use App\Image\ImageHandlerInterface;
+use App\ModelFunctions\MOVFormat;
+use App\Models\Logs;
+use App\Models\Photo;
+use App\Models\SizeVariant;
+use FFMpeg\FFMpeg;
+
+class AddStandaloneStrategy extends AddBaseStrategy
+{
+	public function __construct(AddStrategyParameters $parameters)
+	{
+		parent::__construct($parameters, new Photo());
+	}
+
+	public function do(): Photo
+	{
+		// Create and save "bare" photo object without size variants
+		$this->hydrateMetadata();
+		$this->photo->public = $this->parameters->public;
+		$this->photo->star = $this->parameters->star;
+		$this->setParentAndOwnership();
+		$this->photo->save();
```

Comment:
Because we need an ID for the photo entity, before we can add `SizeVariant`s to it. Otherwise `SizeVariantFactory::createOriginal` (13 lines below) will fail with a `\LogicException` and the error message `cannot create a size variant for a photo whose id is not yet persisted to DB`.